test/utest.c

```diff
@@ -57,7 +57,7 @@
 #define FRAME_LOGLEVEL              LOG_LEVEL_1
 
 #ifndef _WIN32
-#if defined(__APPLE__)
+#if defined(__APPLE__) || defined(__loongarch64)
```

Comment:
Consider using just `__loongarch__` here, as the code enabled by the switch doesn't seem to care about whether it's 64-bit or not.

Reply:
OK